src/contextmenu.js

```diff
@@ -0,0 +1,472 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
```

Comment:
I think any copyright comments should include `MIT` not `Google LLC`. @ewpatton to confirm that `MIT` is correct.